ci/scripts/assume-role.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const AWS = require("aws-sdk")
```

Comment:
Should probably standardise on single quotes `'` here.

Reply:
👍